### PR TITLE
Add create method for entities

### LINKED ISSUE #1468: Support creation of entities via API
Support creation of entities via API & sensuctl

## Expected Behavior
Support creation of entities via API & sensuctl.

## Current Behavior
Can't create entities (e.g. e.g. proxy entities).

## Possible Solution
N/A

## Steps to Reproduce (for bugs)
N/A

## Context
N/A

## Your Environment
N/A

### PR DESCRIPTION
**Pull request by mercul3s**
## What is this change?

This change adds a create method to the entities API.

## Why is this change necessary?

Closes #1468 

## Does your change need a Changelog entry?

Added.

## Do you need clarification on anything?

Nope!

## Were there any complications while making this change?

Nope!

### CODE REVIEW

File: backend/apid/actions/entities.go [hunk 1]
```diff
@@ -90,6 +91,36 @@ func (c EntityController) Query(ctx context.Context) ([]*types.Entity, error) {
...
+	// Verify viewer can create the resource
+	if yes := abilities.CanCreate(&entity); !yes {
+		return NewErrorf(PermissionDenied)
+	}
+
+	// Check for an already existing resource
+	if e, err := c.Store.GetEntityByID(ctx, entity.ID); err != nil {
+		return NewError(InternalErr, err)
+	} else if e != nil {
+		fmt.Println("e in entity fetch", e)
```
Comment (palourde) [inline backend/apid/actions/entities.go]: Can probably be removed 😛

Comment (mercul3s) [reply backend/apid/actions/entities.go]: 😅 argh I missed one! It's gone now.

**Comment by mercul3s** [comment]
@jamesdphillips lmk if I need to file an issue for graphql entity work, I'm happy to do so if needed but not sure what the description might be as I'm unfamiliar with the implementation.

File: backend/apid/actions/entities.go [hunk 2]
```diff
@@ -90,6 +90,35 @@ func (c EntityController) Query(ctx context.Context) ([]*types.Entity, error) {
...
+	// Verify viewer can create the resource
+	if yes := abilities.CanCreate(&entity); !yes {
+		return NewErrorf(PermissionDenied)
+	}
+
+	// Check for an already existing resource
+	if e, err := c.Store.GetEntityByID(ctx, entity.ID); err != nil {
+		return NewError(InternalErr, err)
+	} else if e != nil {
+		return NewErrorf(AlreadyExistsErr)
```
Comment (oferrigni) [inline backend/apid/actions/entities.go]: It doesn't look like this error case is covered by tests

Comment (mercul3s) [reply backend/apid/actions/entities.go]: Whoops, I can add a test for that.

**Pull request merged**